funding/batch.go

```diff
@@ -329,6 +329,7 @@ func (b *Batcher) BatchFund(ctx context.Context,
 		},
 		MinConfs:         firstReq.MinConfs,
 		SpendUnconfirmed: firstReq.MinConfs == 0,
+		ChangeType:       walletrpc.ChangeAddressType_CHANGE_ADDRESS_TYPE_P2TR, //nolint:lll
```

Comment:
nit: move to above the struct initialization to avoid line too long.
```
	firstReq := b.channels[0].fundingReq
	feeRateSatPerKVByte := firstReq.FundingFeePerKw.FeePerKVByte()
	changeType := walletrpc.ChangeAddressType_CHANGE_ADDRESS_TYPE_P2TR
	fundPsbtReq := &walletrpc.FundPsbtRequest{
		Template: &walletrpc.FundPsbtRequest_Raw{
			Raw: txTemplate,
		},
		Fees: &walletrpc.FundPsbtRequest_SatPerVbyte{
			SatPerVbyte: uint64(feeRateSatPerKVByte) / 1000,
		},
		MinConfs:         firstReq.MinConfs,
		SpendUnconfirmed: firstReq.MinConfs == 0,
		ChangeType:       changeType,
	}
```